src/platform/packages/shared/kbn-test/src/auth/session_manager.ts

```diff
@@ -67,6 +69,10 @@ export class SamlSessionManager {
     };
     this.kbnHost = Url.format(hostOptionsWithoutAuth);
     this.isCloud = options.isCloud;
+    this.cloudHostName = options.cloudHostName || process.env.TEST_CLOUD_HOST_NAME;
```

Comment:
Scout will pass it in constructor, because it is defined in configuration file ( e.g. `mki.json` ). FTR will continue to rely on `TEST_CLOUD_HOST_NAME` to be set locally or CI